va/va.h

```diff
@@ -1916,6 +2014,77 @@ typedef struct _VAContextParameterUpdateBuffer
     uint32_t reserved[VA_PADDING_MEDIUM];
 } VAContextParameterUpdateBuffer;
 
+/** \brief AES CTR encryption type */
+#define VA_ENCRYPTION_TYPE_CENC_CTR             0x00000001
```

Comment:
Is this right? So far we've been using this:

#define VA_ENCRYPTION_TYPE_NONE 0x00000000
#define VA_ENCRYPTION_TYPE_CENC_CBC 0x00000002
#define VA_ENCRYPTION_TYPE_CENC_CTR 0x00000008
#define VA_ENCRYPTION_TYPE_CTR_128 0x00000010
#define VA_ENCRYPTION_TYPE_CBC 0x00000020

Reply:
Good point. We are preparing write-up.